wicker/core/datasets.py

```diff
@@ -131,8 +220,12 @@ def __getitem__(self, idx: int) -> Dict[str, Any]:
             self.schema(),
         )
 
-    def _get_parquet_dir_size(self) -> int:
-        """Get the arrow path and find all the files within, count their bytes
+    def _get_parquet_dir_size(self, entire_dataset_parquet_dir_size: bool = False) -> int:
+        """Get the parquet path and find all the files within, count their bytes
+
+        Args:
+            entire_dataset_parquet_dir_size (bool): Whether to get the entire parquet dir or just
```

Comment:
the naming of this param make it look like an integer than a bool, how about `should_compute_parquet_dir_size`?

Reply:
Good ask, thanks!